guillotina/test_package.py

```diff
@@ -213,6 +213,16 @@ def __init__(self, settings=None, loop=None):
     return request.matchdict
 
 
+@configure.service(
```

Comment:
Can you add tests in case you answer a Response object and a Streamed response object ?